_data/simple-icons.json

```
@@ -8546,7 +8582,8 @@
         {
             "title": "Squarespace",
             "hex": "000000",
-            "source": "http://squarespace.com/brand-guidelines"
+            "source": "https://www.squarespace.com/logo-guidelines",
+            "guidelines": "http://squarespace.com/brand-guidelines"
```

Both for consistency and because it redirects there, I think we should add `www` to the guidelines URL.

```suggestion
            "source": "https://www.squarespace.com/logo-guidelines",
            "guidelines": "http://www.squarespace.com/brand-guidelines"
```

---

Sure, I dont know how I manage that 😅